packages/wrangler/src/api/dev.ts

```diff
@@ -223,74 +219,41 @@ export async function unstable_dev(
 		logLevel: options?.logLevel ?? defaultLogLevel,
 		port: options?.port ?? 0,
 		experimentalVersions: undefined,
-		experimentalDevEnv: devEnv,
+		experimentalDevEnv: undefined,
```

Comment:
could we remove this arg altogether? or would that constitute a breaking change?

Reply:
It wouldn't (since it's experimental), but what we've tended to do in the past is keep around the experimental flag for a bit with a message saying that it's no longer needed, just in case people are still using it. That's what I did in this PR, but absolutely open to changing if you think that'd be better!

Reply:
no no, let's stay consistent in how we do things 👍